test/Integration/broadcast-row.mlir

```diff
@@ -0,0 +1,25 @@
+// RUN: tpp-run --vector-to-XSMM %s -e rowBroadcast --entry-point-result=void -print-mlir=mid 2>&1 | FileCheck %s -check-prefix=ROWBROADCAST
+// RUN: tpp-run %s -e rowBroadcast --entry-point-result=void -print-mlir=mid 2>&1 | FileCheck %s -check-prefix=ROWBROADCAST
+
+
+#map4 = affine_map<(d0, d1) -> (d1, 0)>
+#map5 = affine_map<(d0, d1) -> (d0, d1)>
+
+func.func @rowBroadcast(%arg0: tensor<128x1xf32>, %arg1: tensor<512x128xf32>) -> tensor<512x128xf32> {
```

Comment:
This looks odd. I was expecting `<128x1>` to broadcast to `<128x?>` and not transposed like that.

Reply:
https://github.com/plaidml/tpp-mlir/blob/d7fcc28647aaf3ef853349296139414e856aac61/test/Conversion/LinalgToXsmm/linalg-to-unary.mlir#L280
This is the same behavior as earlier.